kube-ps1.sh

```diff
@@ -31,13 +31,20 @@ KUBE_PS1_PREFIX="${KUBE_PS1_PREFIX-(}"
 KUBE_PS1_SEPARATOR="${KUBE_PS1_SEPARATOR-|}"
 KUBE_PS1_DIVIDER="${KUBE_PS1_DIVIDER-:}"
 KUBE_PS1_SUFFIX="${KUBE_PS1_SUFFIX-)}"
+KUBE_PS1_LAST_TIME=0
+
+# Color customization
+KUBE_PS1_PREFIX_COLOR="${KUBE_PS1_PREFIX_COLOR}"
```

Comment:
what's the point of assigning itself?

Reply:
This was just a POC I was working on to get some thoughts.